utils/python/CIME/case_lt_archive.py

```diff
@@ -12,17 +12,16 @@ def case_lt_archive(case):
     caseroot = case.get_value("CASEROOT")
 
     # max number of threads needed by scripts
-    os.environ["maxthrds"] = 1
+    os.environ["maxthrds"] = "1"
 
     # document start
     append_status("lt_archive starting",caseroot=caseroot,sfile="CaseStatus")
 
     # determine status of run and short term archiving
     runComplete = does_file_have_string(os.path.join(caseroot, "CaseStatus"),
```

Comment:
I'm really concerned about these checks - they are difficult to maintain because status strings they are testing for may change.  But also a single successful run in the case will make these true for all future runs and so they don't really provide anything useful for a long running case - I suggest they be removed and we find another method.